local-cli/link/windows/patches/makeProjectPatch.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function makeProjectPatch(projectInsert, projectName = '') {
+  return {
+    pattern: '<ProjectReference Include="..\\..\\node_modules\\react-native-windows\\ReactWindows\\ReactNative\\ReactNative.csproj">',
```

Comment:
Nevermind, you use the `flip` parameter to make sure the patch comes before the ProjectReference.

---
In reply to: [90884760](https://github.com/facebook/react-native/pull/11282#discussion_r90884760) [](ancestors = 90884760)